src/StackExchange.Exceptional.AspNetCore/ExceptionalBuilderExtensions.cs

```diff
@@ -32,11 +34,10 @@ public static IApplicationBuilder UseExceptional(this IApplicationBuilder builde
         /// <param name="builder">The <see cref="IApplicationBuilder"/> instance this method extends.</param>
         /// <param name="applicationName">Application name for this error log.</param>
         /// <exception cref="ArgumentNullException"><paramref name="builder"/> or <paramref name="applicationName"/> is <c>null</c>.</exception>
-        public static IApplicationBuilder UseExceptional(this IApplicationBuilder builder, string applicationName)
+        public static IApplicationBuilder UseExceptional(this IApplicationBuilder builder, IConfiguration configuration, string applicationName)
```

Comment:
Let's see if we can do this cleaner, e.g. can IConfiguration be injected or accessed another way? Passing it in seems very odd here, pretty sure there's a way around (I can take a stab this week)